src/js/components/Select/SelectContainer.js

```diff
@@ -116,7 +116,15 @@ class SelectContainer extends Component {
         nextValue = [];
         nextSelected = [];
         let removed = false;
-        (selected || []).forEach((selectedIndex) => {
+        let selectedValuesIx = [];
```

Comment:
what `Ix` stands for?

can we use just `selectedValues` or `selectedIndexes` ?